Student.java

```diff
@@ -0,0 +1,45 @@
+public class Student {
+    private static String name;
+    private static int course;
+    private static int grade;
+
+
+    public String getName() {
+        return name;
+    }
+
+    public int getCourse() {
+        return course;
+    }
+
+    public int getGrade() {
+        return grade;
+    }
+
+    public void setName(String name) {
```

Comment:
According to the home task there is restriction for name (not less than 3). Add this check .
if (name.length() > 3) {
            this.name = name;